_sass/profile.scss

```
@@ -1,5 +1,6 @@
 .profile {
-    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.05), black), no-repeat center/100% url('/assets/img/background.jpg'), #946c50;
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.05), black),
+        no-repeat center/100% url("/assets/img/background.jpg"), #946c50;
```

```suggestion
    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.05), black), no-repeat center/100% url('/assets/img/background.jpg'), #946c50;
```
Don't change this unnecessary to avoid merge conflicts.

---

Oh no, I didn't change that. It's prolly my editor's code formatting.